lib/generators/blacklight/solr5_generator.rb

```diff
@@ -16,6 +23,14 @@ def install_solrwrapper
       append_to_file "Rakefile", "\nrequire 'solr_wrapper/rake_task' unless Rails.env.production?\n"
     end
 
+    def copy_solr_conf
+      directory 'solr', 'solr'
+    end
+
+    def solr_wrapper_config
+      copy_file '.solr_wrapper', '.solr_wrapper'
```

Comment:
Can we please rename this file `.solr_wrapper.yml`? It was ok just in blacklight, but I don't want to propagate it downstream.